deploy/docker-setup.sh

```diff
@@ -32,9 +32,11 @@ edumfa-manage db upgrade -d /usr/local/lib/edumfa/migrations
 echo "Creating Admin"
 edumfa-manage admin add $EDUMFA_ADMIN_USER -p $EDUMFA_ADMIN_PASS
 
-echo "Executing User-Scripts"
 # Execute user scripts
-bash /opt/edumfa/user-scripts/*.sh
+if [ "$(ls -A /opt/edumfa/user-scripts/)" ]; then
```

Comment:
Mhm. I considered using: 
`if compgen -G "/opt/edumfa/user-scripts/*.sh" > /dev/null; then `

Reply:
Yes this works thanks. I updated the branch. Are there any advantages from using compgen or is it because it's a builtin ?

Reply:
compgen allows to find shell scripts. So in case there are any other files like a README.md or something like that it prop. would break in your case...

Reply:
Ah yes thats true. Didn't think someone would place anything else there.